source/core/src/main/com/csse3200/game/components/inventory/PlayerInventoryDisplay.java

```diff
@@ -207,6 +248,26 @@ void generateHotBar() {
         stage.addActor(hotBarDisplay);
     }
 
+
+    private ImageButton createSlot(int index) {
+        ImageButton slot = new ImageButton(slotSkin);
+        AbstractItem item = inventory.getAt(index);
+        if (item != null) {
+            addSlotListeners(slot, item, index);
+            Image itemImage = new Image(new Texture(item.getTexturePath()));
+            slot.add(itemImage).center().size(70, 70);
+            // Add a label for item quantity (subscript)
+            int itemCount = item.getQuantity(); // Assuming there's a method for getting item count
```

Comment:
Comment for assumption is unnecessary - the base AbstractItem class already has this functionality.